src/main/java/core/basesyntax/model/RightTriangle.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax.model;
+
+public class RightTriangle extends Color {
+    private double firstLeg;
+    private double secondLeg;
+
+    public RightTriangle(String color, double firstLeg, double secondLeg) {
```

Comment:
The constructor should not begin with an empty line. Please remove the empty line above the constructor declaration.